views/app.njk

```diff
@@ -25,5 +25,8 @@
 {% block main_content %}
   <div id="js-app"></div>
 
+  <script>
+    window.CSRF_TOKEN = "{{ csrfToken }}";
```

Comment:
expose the CSRF token as a global var so that it can be grabbed by our `fetch` implementation